alibi_detect/utils/loading.py

```diff
@@ -0,0 +1,50 @@
+# type: ignore
```

Comment:
Does this do anything?

Reply:
Not right now, I'll remove. As things stand it will be required again in part III, but I will have another think about whether things can be rewritten there so it is not required.